OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -185,6 +213,18 @@ private bool IsFilterValid(WorkshopFilter filter)
             return filter != null && filter.MaxStartTime >= filter.MinStartTime
                                   && filter.MaxAge >= filter.MinAge
                                   && filter.MaxPrice >= filter.MinPrice;
+        } 
+        
+        public async Task<IEnumerable<string>> GetNotificationsRecipientIds(NotificationAction action, Dictionary<string, string> additionalData, Guid objectId)
+        {
+            var recipientIds = new List<string>();
+
+            var usersInFavoriteWorkshop = await favoriteService.GetAll().ConfigureAwait(false);
```

Comment:
This call is too heavy for the DB. Please use filter here.

Reply:
Please ask what filter should I use?